app/Http/Requests/StoreOrderRequest.php

```diff
@@ -23,16 +23,30 @@ public function authorize()
      */
     public function rules()
     {
-        return [
-            'quantity' => 'required|integer|min:1|max:10000'
-        ];
+        if ($this->has('quantity'))
+        {
+            return [
+                'quantity' => 'required|integer|min:1|max:10000',
+            ];
+        }else{
+            return [
+                'price' => 'required|integer|min:1|max:10000',
```

Comment:
o tai jei aš tiesiog užklausoj siųsiu price, o ne quantiy, ar kaina nepasikeis? turėtų authorize numatyt šitą reikalą